channels/ir.m3u

```diff
@@ -33,25 +33,39 @@ https://bozztv.com/36bay2/gin/giniko_tvbox-cine-series_800kb_36bay2/tracks-v1a1/
 https://d2e40kvaojifd6.cloudfront.net/stream/24b/chunklist_w1808578907.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Bollywood" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-bollywood.png?1212" group-title="Entertainment",GEM Bollywood
+#EXTINF:-1 tvg-id="" tvg-name="GEM Bollywood" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gembollywood.png?13" group-title="Entertainment",GEM Bollywood
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_bollywood/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Classic" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-classic.png?1212" group-title="Entertainment",GEM Classic
+#EXTINF:-1 tvg-id="" tvg-name="GEM Classic" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemclassic.png?13" group-title="Entertainment",GEM Classic
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_classic/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Comedy" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-comedy.png?1212" group-title="Comedy",GEM Comedy
+#EXTINF:-1 tvg-id="" tvg-name="GEM Comedy" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemcomedy.png?13" group-title="Comedy",GEM Comedy
 https://d2e40kvaojifd6.cloudfront.net/stream/gem_comedy/chunklist_w1019651088.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Drama" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-drama.png?1212" group-title="Movies",GEM Drama
+#EXTINF:-1 tvg-id="" tvg-name="GEM Drama" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemdrama.png?13" group-title="Movies",GEM Drama
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_drama/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Junior" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-junior.png?1212" group-title="Entertainment",GEM Junior
+#EXTINF:-1 tvg-id="" tvg-name="GEM Film" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_film.png?13" group-title="Entertainment",Gem Film
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_film/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Fit" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_fit.png?13" group-title="Entertainment",Gem Fit
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_fit/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Food" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_food.png?13" group-title="Entertainment",Gem Food
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_food/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Junior" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemjunior.png?13" group-title="Entertainment",GEM Junior
```

Comment:
Links do not work.

Reply:
I just checked with https://castr.io/hlsplayer and they are working